parodos-model-api/src/main/java/com/redhat/parodos/workflow/parameter/WorkFlowParameter.java

```diff
@@ -43,12 +43,12 @@ public class WorkFlowParameter {
 
 	private boolean optional;
 
-	public Map<String, String> getAsJsonSchema() {
+	public Map<String, Object> getAsJsonSchema() {
 		if (this.getType() == null) {
 			return Map.of();
 		}
-		Map<String, String> properties = this.getType().getAsJsonSchema();
-		properties.put("required", String.valueOf(!this.isOptional()));
+		Map<String, Object> properties = this.getType().getAsJsonSchema();
+		properties.put("required", !this.isOptional());
```

Comment:
Doubt: did you check if it's ok to put a `boolean` instead of a `Boolean` if the map expect `Object`?

Reply:
I am not sure Java itself allows primitive types in Collections, but it may have change